posthog/tasks/exporter.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import os
+import time
+import uuid
+
+import structlog
+from django.conf import settings
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from webdriver_manager.chrome import ChromeDriverManager
+from webdriver_manager.utils import ChromeType
+
+from posthog.celery import app
+from posthog.models.exported_asset import ExportedAsset
+
+logger = structlog.get_logger(__name__)
+
+TMP_DIR = "/tmp"  # NOTE: Externalise this to ENV var
```

Comment:
do we use object storage instead of TMP 🤔 ?

Reply:
Selenium has to have a place to save the file - hence `/tmp` as once we have loaded it and stored it in PG (in the future S3) we delete the tmp file that was created